Podfile

```diff
@@ -104,6 +104,10 @@ target 'WordPress' do
     pod 'Starscream', '3.0.6'
     pod 'ZendeskSDK', '2.2.0'
 
+    ## Screenshot Generation Helper
+    ## ====================
+    ##
+    pod 'SimulatorStatusMagic', :configurations => ['Debug']
```

Comment:
Is it possible for the pod to be added to the `XCUITest` target itself? I know the target doesn't have access o the app, but perhaps it can still modify the simulator. If at all possible it would be nice to avoid adding code to the app purely for the purpose of testing.

Reply:
Did you investigate adding this to the `XCUITest` target?